src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -35,6 +35,9 @@
 using Roslyn.Utilities;
 using SVsServiceProvider = Microsoft.VisualStudio.Shell.SVsServiceProvider;
 
+// This assembly is currently broken due to interop limitations. Force the runtime to reject it.
```

Comment:
📝 All changes to this file will be reverted when compatible NuGet packages are available. Until then, this assembly is forced into an unusable state.

Reply:
➡️ These changes are now reverted in #52262